content/fr/docs/setup/_index.md

```diff
@@ -0,0 +1,93 @@
+---
+reviewers:
+- brendandburns
+- erictune
+- mikedanese
+no_issue: true
+title: Setup
+main_menu: true
+weight: 30
+content_template: templates/concept
+---
+{{% capture overview %}}
+
+Utilisez cette page pour trouver le type de solution qui correspond le mieux à vos besoins.
+
+Le choix de l'emplacement de Kubernetes dépend des ressources dont vous disposez 
+et de la flexibilité dont vous avez besoin. Vous pouvez executer Kubernetes presque partout, 
+de votre ordinateur portable aux machines virtuelles d'un fournisseur de cloud jusqu'à un rack de serveurs en bare metal.
+Vous pouvez également mettre en place un cluster entièrement géré en exécutant une seule commande ou bien créer 
+votre propre cluster personnalisé sur vos serveurs bare-metal.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Solutions pour machine locale
+
+La solution machine locale est un moyen facile de démarrer avec Kubernetes. Vous
+pouvez créer et tester des clusters Kubernetes sans vous soucier de la consommation 
+des ressources et des quotas d'un cloud.
+
+Vous devriez choisir une solution locale si vous souhaitez :
+
+* Essayer ou commencer à apprendre Kubernetes
+* Développer et réaliser des tests sur des clusters locaux
+
+Choisissez une [solution machine locale] (/docs/setup/pick-right-solution/#local-machine-solutions).
```

Comment:
Correction (no space) `[solution machine locale](/docs/setup/pick-right-solution/#local-machine-solutions)` otherwise the link will be broken.

Reply:
same result in `v0.54.0`. Ignore my comment.

Reply:
seems in the current version of hugo, it doesn't break the link. testing it locally and online. and rendering seems better